app/src/main/java/nl/quantifiedstudent/watch/binary/VarInt.kt

```diff
@@ -0,0 +1,285 @@
+package nl.quantifiedstudent.watch.binary
+
+import nl.quantifiedstudent.watch.extensions.shl
+import java.io.IOException
+import java.io.InputStream
+import java.io.OutputStream
+import java.nio.ByteBuffer
+
+
+object VarInt {
```

Comment:
Every function is well documented. But the name of the object and the function names are just very ambiguous. The thing a function does needs a description in the name.

Reply:
This class is taken from another API, which perhaps should be better documented. I will create my own implementation for this class to circumvent the ambiguity.